src/features/user/MFV2/InfoAndCTA/DonorCircleInvitation/index.tsx

```diff
@@ -1,10 +1,54 @@
 import React from 'react';
 import styles from '../InfoAndCta.module.scss';
+import DonorCircleInvitationHeader from './DonorCircleInvitationHeader';
+import SingleInvitationCard from './SingleInvitationCard';
+import { useTranslations } from 'next-intl';
 
 const DonorCircleInvitation = () => {
+  const t = useTranslations('Profile');
```

Comment:
As a whole, name this as `PublicProfileActions`. The `DonorCircleInvitation` refers to only the Become a Member button